aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js

```diff
@@ -0,0 +1,29 @@
+import { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { Box, Text } from 'grommet';
+
+// mimics Grommet's FormField label in terms of accessibility considerations
+// by including as="label", htmlFor, and name
+// applies NameValueList text styling
+export const NameValueListFormLabel = ({ data, name }) => {
+  const theme = useContext(ThemeContext);
+  return (
+    <Box margin={theme.formField.label.margin}>
```

Comment:
What do you think about this? I think the alignment between the label and input value is better. (Sorry for providing conflicting feedback on this one 😬 )

```suggestion
    <Box margin={{ top: 'small' }} >
```

**Before**
![Screen Shot 2021-12-16 at 3 06 15 PM](https://user-images.githubusercontent.com/1756948/146455990-a5af515f-9529-4438-8b14-f63b30eab078.png)


**After (suggestion applied)**
![Screen Shot 2021-12-16 at 3 05 58 PM](https://user-images.githubusercontent.com/1756948/146455983-82d0314c-ba2b-40a5-8d96-c4c9c03e7d6d.png)